.circleci/config.yml

```diff
@@ -52,6 +59,10 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - lint-1.13:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
Got curious and checked why we have that: https://circleci.com/docs/2.0/configuration-reference/#tags

It seems to be to run on tags too :)